File: app/src/main/java/com/artmurka/artmurkaapp/Presenter/Adapters/PagerAdapter.java
package com.artmurka.artmurkaapp.Presenter.Adapters;

import android.app.Fragment;
import android.app.FragmentManager;
import android.os.Bundle;
import android.support.v13.app.FragmentStatePagerAdapter;

import com.artmurka.artmurkaapp.Views.Fragments.FragmentAboutGoods;
import com.artmurka.artmurkaapp.Views.Fragments.FragmentDescriptionGoods;


public class PagerAdapter extends FragmentStatePagerAdapter {

    private int numberOfTabs;
    private String currentId;
    private Bundle bundle;
    public PagerAdapter(FragmentManager fm, int numberOfTabs, String id) {
        super(fm);
        this.numberOfTabs = numberOfTabs;
        this.currentId = id;
        bundle = new Bundle();
        bundle.putString("id", currentId);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position){
            case 0:
                FragmentAboutGoods fragmentAboutGoods = new FragmentAboutGoods();
                fragmentAboutGoods.setArguments(bundle);
                return fragmentAboutGoods;
            case 1:
                FragmentDescriptionGoods fragmentDescriptionGoods = new FragmentDescriptionGoods();
                fragmentDescriptionGoods.setArguments(bundle);
                return fragmentDescriptionGoods;
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return numberOfTabs;
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/presenter/adapters/PagerAdapter.kt
package com.artmurka.artmurkaapp.presenter.adapters

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter
import com.artmurka.artmurkaapp.android.views.fragments.FragmentAboutGoods
import com.artmurka.artmurkaapp.android.views.fragments.FragmentDescriptionGoods


class PagerAdapter(fm: FragmentManager, private val numberOfTabs: Int, private val currentId: String) : FragmentStatePagerAdapter(fm) {
    private val bundle: Bundle

    init {
        bundle = Bundle()
        bundle.putString("id", currentId)
    }

    override fun getItem(position: Int): Fragment? {
        when (position) {
            0 -> {
                val fragmentAboutGoods = FragmentAboutGoods()
                fragmentAboutGoods.arguments = bundle
                return fragmentAboutGoods
            }
            1 -> {
                val fragmentDescriptionGoods = FragmentDescriptionGoods()
                fragmentDescriptionGoods.arguments = bundle
                return fragmentDescriptionGoods
            }
            else -> return null
        }
    }

    override fun getCount(): Int {
        return numberOfTabs
    }
}
